<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWidget</class>
 <widget class="QWidget" name="AuthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>505</width>
    <height>141</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="authButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pinInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="cardNumberLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Card Number:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pinLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PIN:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="authMessage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Please, provide the Card Number and PIN-code</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="cadNumberInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
